packages/design-toolkit/src/components/menu/styles.module.css

```diff
@@ -0,0 +1,417 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+@reference '../../index.css';
+
+@layer components.l1 {
+  .menu {
+    @apply rounded-medium bg-surface-overlay shadow-elevation-overlay outline-static overflow-x-clip overflow-y-auto outline;
+  }
+
+  .item {
+    @apply gap-x-s px-s text-body-s fg-primary-bold relative grid cursor-pointer items-center outline outline-transparent;
+    grid-template-columns: auto auto 1fr auto;
+    grid-template-areas: 'icon label space action' 'icon description space action';
+
+    &:before {
+      @apply pointer-events-none absolute top-0 left-0 h-full w-xs;
+    }
+
+    &.cozy {
+      @apply pt-s pb-s;
+    }
+
+    &.compact {
+      @apply pt-xs pb-xs;
+    }
+
+    @variant color-info {
+      @variant focus-visible {
+        @apply bg-accent-primary-muted;
+      }
+
+      @variant hover {
+        @apply bg-accent-primary-muted;
+      }
+
+      @variant open {
+        @apply bg-accent-primary-muted;
+      }
+
+      @variant pressed {
+        @apply fg-accent-primary-bold bg-accent-primary-pressed;
+      }
+
+      @variant selected {
+        @apply bg-accent-primary-muted;
+
+        &:before {
+          @apply bg-accent-primary-bold;
+        }
+
+        @variant hover {
+          @apply fg-a11y-on-accent bg-accent-primary-bold;
+
+          &:before {
+            /* note: i also see this bg-(--outline-accent-primary-pressed). which one is correct? */
+            @apply bg-(--outline-accent-primary-hover);
```

Comment:
to help review: this is
`menu item -> color info -> selected -> hover -> :before`

what is the correct color?